app/views/capacity.html

```diff
@@ -137,7 +137,7 @@ <h3 class="heading-medium">Capacity breakdown</h3>
           <td headers="paroms breakdown">-</td>
           <td headers="fdrs breakdown">-</td>
           <td headers="sdrs breakdown">-</td>
-          <td headers="arms breakdown">-</td>
+          <td headers="arms breakdown">{{ member.armsTotalCases }}</td>
```

Comment:
should this not be rounded too? `{{ member.armsTotalCases | round(1) }}`

Reply:
This is an integer and therefore shouldn't need to be rounded.